src/handlers/ExecuteViewSubmitHandler.ts

```diff
@@ -150,15 +177,196 @@ export class ExecuteViewSubmitHandler {
 			this.read.getPersistenceReader(),
 			user.id,
 		);
-
-		if (!isSupportedLanguage(LanguageInput)) {
-			return this.context.getInteractionResponder().errorResponse();
-		}
 		await userPreference.storeUserPreference({
 			userId: user.id,
-			language: LanguageInput,
+			language: languageInput,
 		});
 
 		return this.context.getInteractionResponder().successResponse();
 	}
+
+	private async handleSend(
+		room: IRoom,
+		user: IUser,
+		view: IUIKitSurface,
+	): Promise<IUIKitResponse> {
+		try {
+			let body = '';
+
+			const replyCacheStorage = new CacheReplyStorage(
+				this.persistence,
+				this.read.getPersistenceReader(),
+			);
+			const cachedReply = await replyCacheStorage.getCacheReply(user);
+			const bodyStateValue = view.state?.[
+				sendModalEnum.REPLY_BODY_BLOCK_ID
+			]?.[sendModalEnum.REPLY_BODY_ACTION_ID] as string;
+
+			console.log(bodyStateValue);
+
+			body = bodyStateValue ? bodyStateValue : cachedReply.body;
+
+			const message = body.trim();
+			if (!message) {
+				return this.context.getInteractionResponder().errorResponse();
+			}
+
+			await sendMessage(this.modify, user, room, body);
+			await replyCacheStorage.removeCacheReply(user);
+			return this.context.getInteractionResponder().successResponse();
+		} catch (error) {
+			console.error('Error handling send:', error);
+			return this.context.getInteractionResponder().errorResponse();
+		}
+	}
+
+	private async handleDelete(
+		room: IRoom,
+		user: IUser,
+		view: IUIKitSurface,
+		language: Language,
+	): Promise<IUIKitResponse> {
+		console.log('deleteHanler ');
+		const persistenceRead = this.read.getPersistenceReader();
+		const replyStorage = new ReplyStorage(
+			this.persistence,
+			persistenceRead,
+		);
+
+		const replyCacheStorage = new CacheReplyStorage(
+			this.persistence,
+			this.read.getPersistenceReader(),
+		);
+		const cachedReply = await replyCacheStorage.getCacheReply(user);
+		console.log(cachedReply);
+
+		const replyId = cachedReply.id;
+		const result = await replyStorage.deleteReplyById(
+			user,
+			replyId,
+			language,
+		);
+
+		if (result.success) {
+			const successMessage = `${t('quick_reply', language)} **${
+				cachedReply.name
+			}** ${t('delete_successfully', language)}✅`;
```

Comment:
This happens in other places too, so consider changing in all occurrences... 

When you want to display a composable message, don't "merge" i18n keys, some languages have different orders of sentences

I'd suggest using variables interpolations https://react.i18next.com/latest/trans-component#interpolation

Reply:
You can also use the <Trans> component

Reply:
we can also use the` __parameter__ `to insert the parameters